Index: app/node_modules/active-win/lib/linux.js
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/node_modules/active-win/lib/linux.js
+++ app/node_modules/active-win/lib/linux.js
@@ -7,8 +7,8 @@
 const readFile = promisify(fs.readFile);
 const readlink = promisify(fs.readlink);

-const xpropBin = 'xprop';
-const xwininfoBin = 'xwininfo';
+const xpropBin = __dirname + '/../../../natives/linux/xprop';
+const xwininfoBin = __dirname + '/../../../natives/linux/xwininfo';
 const xpropActiveArgs = ['-root', '\t$0', '_NET_ACTIVE_WINDOW'];
 const xpropDetailsArgs = ['-id'];

@@ -49,7 +49,7 @@
 	return {
 		platform: 'linux',
 		title: JSON.parse(result['_NET_WM_NAME(UTF8_STRING)'] || result['WM_NAME(STRING)']) || null,
-		id: windowId,
+		id: activeWindowId,
 		owner: {
 			name: JSON.parse(result['WM_CLASS(STRING)'].split(',').pop()),
 			processId
